config/locales/en.yml

```diff
@@ -106,6 +106,8 @@ en:
       Terms of Use describes the conditions under which content may be used. This is an optional field.
     physical_description: |
       Physical Description is an optional field that will display information about the original resource being described.
+    rights_statement: |
+      RightsStatement.org rights statement
```

Comment:
@jlhardes @jenyoung Could you two please write the help text that will appear above the rights statement field on the resource description page?